obsei/sink/base_sink.py

```diff
@@ -43,10 +43,7 @@ class BaseSink(BaseModel):
 
     @abstractmethod
     def send_data(
-        self,
-        analyzer_responses: List[TextPayload],
-        config: BaseSinkConfig,
-        **kwargs
+        self, analyzer_responses: List[TextPayload], config: BaseSinkConfig, **kwargs
```

Comment:
Bit strange, on my system black do reverse 👀

Reply:
yeah, I will check it once again

Reply:
I'm using the latest black version 21.6b0

Reply:
okay